app/lib/nourish/stripe.rb

```diff
@@ -0,0 +1,15 @@
+module Nourish
+  # Nourish Stripe plugin for accepting payments via stripe
+  class Stripe
+    def create_charge(*args, **kwargs)
+      ::Stripe::Charge.create(*args, **kwargs)
+    end
+
+    class << self
+      attr_writer :instance
```

Comment:
Is there a reason we can't use Ruby's baked in `Singleton` stuff here? Feels like clearer intent.

Reply:
Not really, I had some bad experiences with ruby singleton before and have shied away from it since, but it doesn't really matter to me.